docs/fargate.md

```diff
@@ -0,0 +1,438 @@
+  --region "${AWS_REGION}"
+
+export EFS_ID="$(
+  aws efs describe-file-systems \
+    --region="${AWS_REGION}" | \
+    jq '.FileSystems[] |
+      select(.Tags[].Key == "Name" and .Tags[].Value == "SumologicCollectionVolumes") |
+      .FileSystemId' \
+      --raw-output)"
+```
+
+The following snippet ensures that the Volume won't be duplicated:
+
+```bash
+## Create EFS Volume
+export EFS_ID="$(
+  aws efs describe-file-systems \
+    --region="${AWS_REGION}" | \
+    jq '.FileSystems[] |
+      select(.Tags[].Key == "Name" and .Tags[].Value == "SumologicCollectionVolumes") |
+      .FileSystemId' \
+      --raw-output)"
+
+if [[ -z "${EFS_ID}" ]]; then
+  aws efs create-file-system \
+    --encrypted \
+    --performance-mode generalPurpose \
+    --throughput-mode bursting \
+    --tags Key=Name,Value=SumologicCollectionVolumes \
+    --region "${AWS_REGION}"
+  export EFS_ID="$(
+    aws efs describe-file-systems \
+      --region="${AWS_REGION}" | \
+      jq '.FileSystems[] |
+        select(.Tags[].Key == "Name" and .Tags[].Value == "SumologicCollectionVolumes") |
+        .FileSystemId' \
+        --raw-output)"
+fi
+```
+
+`EFS_ID` is going to be used in setting up Volumes for metric pods.
+
+## Metrics
+
+You can install metrics with or without persistence. See the following sections for more details:
+
+- [Persistence disabled](#persistence-disabled)
+- [Persistence enabled](#persistence-enabled)
+
+### Persistence Disabled
+
+To disable persistence, add the following configuration to `user-values.yaml`:
+
+```yaml
+metadata:
+  persistence:
+    enabled: false
+```
+
+**Note: This is going to disable persistence for both logs and metrics.**
+
+### Persistence Enabled
+
+If you want to keep persistence (which is default configuration), you need to manually create Volumes for the Metric Pods. We recommend to
+create Persistent Volume Claims on the top of EFS Storage. In order to set up them, please apply the following steps:
+
+#### Create EFS access points for metric Pods
+
+EFS Access Point is an entry point for an application. We recommend to create Acess Points pointing to the directories using
+`/${NAMESPACE}/${PVC_NAME}` schema.
+
+You can create them using the following `bash` script:
+
+```bash
+## Create EFS access points for metric Pods
+for (( counter=0; counter<"${METRIC_PODS}"; counter++ )); do
+  FSAP_ID="$(
+    aws efs describe-access-points \
+      --region "${AWS_REGION}" |
+    jq ".AccessPoints[] |
+      select(.RootDirectory.Path == \"/sumologic/file-storage-${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics-${counter}\") |
+      .AccessPointId" \
+      --raw-output)"
+
+  if [[ -z "${FSAP_ID}" ]]; then
+    aws efs create-access-point \
+        --file-system-id "${EFS_ID}" \
+        --posix-user Uid=1000,Gid=1000 \
+        --root-directory "Path=/${NAMESPACE}/file-storage-${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics-${counter},CreationInfo={OwnerUid=1000,OwnerGid=1000,Permissions=777}" \
```

Comment:
don't we need to define access points for the paths below used in our helm charts?
- /conf
- /etc/otel
- /var/lib/storage/otc
- /tmp
- /etc/terraform
- /customer-scripts